wiki/fr/fr-Software-Manual.md

```diff
@@ -0,0 +1,226 @@
+
+Vous pouvez définir manuellement la taille du tampon de gigue pour le client local et le serveur distant. Pour la mémoire tampon de gigue locale, les pertes de flux audio sont indiquées par le voyant lumibaux situé sous les curseurs de taille de la mémoire tampon de gigue.
+Si le voyant devient rouge, un dépassement ou une sous-utilisation de la mémoire tampon a eu lieu et le flux audio est interrompu.
+
+Le réglage du tampon de gigue est donc aussi un compromis entre la qualité audio et le délai global.
+
+Si le paramètre Auto est activé, les tampons de gigue du client local et du serveur distant sont automatiquement réglés en fonction des mesures de la gigue de synchronisation du réseau et de l'interface audio. Si la vérification automatique est activée, les curseurs de la taille des tampons de gigue sont désactivés (ils ne peuvent pas être déplacés avec la souris).
+
+### Canaux Audio
+
+![Canaux Audio](https://user-images.githubusercontent.com/20726856/97362020-b24a9b80-18a0-11eb-8a55-ee536b29460b.png)
+
+Sélectionne le nombre de canaux audio à utiliser pour la communication entre le client et le serveur. Trois modes sont disponibles :
+
+Les modes Mono et Stéréo utilisent respectivement un et deux canaux audio.
+
+Entrée Mono/Sortie Stéréo : Le signal audio envoyé au serveur est mono mais le signal de retour est stéréo. C'est utile si l'interface audio a l'instrument sur un canal d'entrée et le micro sur l'autre. Dans ce cas, les deux signaux d'entrée peuvent être mélangés sur un canal mono mais le mixage du serveur est entendu en stéréo.
+
+L'activation du mode stéréo augmentera le débit de votre flux. Assurez-vous que votre débit ne dépasse pas la bande passante disponible pour votre connexion internet.
+
+En mode stéréo, aucune sélection de canal audio pour l'effet de réverbération ne sera disponible dans la fenêtre principale puisque l'effet est alors appliqué aux deux canaux du signal stéréo dans ce cas.
+
+### Qualité audio
+
+![Qualité audio](https://user-images.githubusercontent.com/20726856/97362080-c42c3e80-18a0-11eb-87ac-9a4ca948bbcf.png)
+
+Plus la qualité audio est élevée, plus le débit de données de votre flux audio est élevé. Assurez-vous que votre débit ne dépasse pas la bande passante disponible pour votre connexion internet.
+
+### Niveau de nouveau client
+
+![Niveau de nouveau client](https://user-images.githubusercontent.com/20726856/97362173-e2923a00-18a0-11eb-8401-9ad71866f6b1.png)
+
+Ce paramètre définit le niveau du canal d'un client nouvellement connecté, en pourcentage. Si un nouveau musicien se connecte au serveur, il obtiendra le niveau initial spécifié si aucun autre niveau de canal provenant d'une connexion précédente de ce même client n'a déjà été enregistré. Vous pouvez configurer tous les clients d'un serveur occupé à ce niveau en utilisant Éditer > « Régler tous les charriots sur niveau d'un nouveau client ».
```

Comment:
* "_le niveau du canal_" -> "_le niveau du chariot_"
* "_se connecte au serveur_" -> "_se connecte au serveur courant _"
* "_le niveau initial_" -> "_le niveau de chariot initial_"
* "_été enregistré_" -> "_été stocké_"
* "_charriots_" -> "_chariots_" (my bad, the GUI French translation was wrong on this one, I've fixed it now)

Reply:
Chariots -> Potentiomètreq in the French UI, please! ;)